### ISSUE TITLE
feature request: check all packages in a file, not just the "main" package

### ISSUE THREAD

**Issue by karenetheridge** [plicease/Test-Version#4]
If a file declares multiple packages, there is no check that all the versions are consistent:

```
package Foo;
our $VERSION = '0.002';

package Foo::Bar;
our $VERSION = '0.001';  # oops someone forgot to increment this version
```

...and no check that all packages _have_ a $VERSION declared.

These can be extracted thusly:

```
my $info = Module::Metadata->new($filename);
my $main_version = $info->version;
my $main_package = $info->name;
foreach my $package ($info->packages_inside) {
    ok(defined($info->version($package), "$package has a \$VERSION");
    is($info->version($package), $main_version, "$package version matches version for $main_package ($main_version)");
}
```

I'm not opposed to new options being added to enable either of these checks.

**Comment by karenetheridge** [comment]
(edited main description since the first mail notification)

**Comment by plicease** [comment]
I do think this should be an option so as to avoid breaking existing tests.

**Comment by plicease** [comment]
I added a `multiple` option in 2.01_01.   If it looks good in a couple of days I will do a prod release and add the option to the `Dist::Zilla` plugin.

**Comment by plicease** [comment]
I believe this to be addressed in version 2.02.

**Comment by karenetheridge** [comment]
thanks! - I haven't had a chance to look yet, but I plan to do so soon!
